### PR TITLE
Figure 5

### PR DESCRIPTION
**Pull request by jclagarde**
final part of issue #230

### CODE REVIEW

File: lineage/figures/figure5.py [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
+def accuracy_increased_cells():
+    """
+    Calculates accuracy and transition rate estimation over an increasing number of cells in a lineage for an pruned two-state model.
+    """
+
+    # Creating a list of populations to analyze over
+    list_of_populations = []
+    
+    # Create a list of transition matrices to transition over called list_of_Ts
+    list_of_Ts = []
```
Comment (shak360) [inline lineage/figures/figure5.py]: You need to call your `makeTs` function and store the result in `list_of_Ts`

File: lineage/figures/figure5.py [hunk 2]
```diff
@@ -1,14 +1,67 @@
...
+    
+    # Iterate through the transition matrices
+    for T in list_of_Ts:
+        # Creating an unpruned and pruned lineage
+        lineage = LineageTree(piiii, T, E, (2**12) - 1, experiment_time, prune_condition='both', prune_boolean=True)
```
Comment (shak360) [inline lineage/figures/figure5.py]:
```suggestion
        lineage = LineageTree(piiii, T, E, (2**12) - 1, experiment_time=1e9, prune_condition='both', prune_boolean=True)
```

File: lineage/figures/figure5.py [hunk 3]
```diff
@@ -1,14 +1,67 @@
...
+
+        while len(lineage.output_lineage) < 16:
+            del lineage
+            lineage = LineageTree(piiii, T, E, (2**12) - 1, experiment_time, prune_condition='both', prune_boolean=True)
```
Comment (shak360) [inline lineage/figures/figure5.py]:
```suggestion
            lineage = LineageTree(piiii, T, E, (2**12) - 1, experiment_time=1e9, prune_condition='both', prune_boolean=True)
```

**Comment by shak360** [comment]
image

@aarmey Figure 5 is done.

**Comment by shak360** [comment]
I'll make an even more common figure4Common function in a new branch and change figure 6 with it. This PR will is also the last of JC's assignments in #230

**Comment by aarmey** [comment]
The code all looks good. Any ideas about what's happening with the transition matrix estimation?

**Comment by shak360** [comment]
Hmmmm, The Initial probability matrix estimation I'm not worried about because these are all single lineages so it doesn't make sense to even plot those.

In a similar scenario:
image

(with normal transitions instead of transitions going from 0.5 to 0 or 0.5 to 1) this is what we get

**Pull request merged**